poppy/wfe.py

```diff
@@ -323,3 +324,32 @@ def get_opd(self, wave, units='meters'):
         if units == 'waves':
             opd /= wave.wavelength.to(u.meter).value
         return opd
+
+
+class StatsticalPSDWFE(WavefrontError):
+    """
+    Statistical PSF WFE class from power law for optical noise.
+
+    Input WFE in radians RMS.
+    """
+
+    @utils.quantity_input(wfe=u.rad)
+    def __init__(self, name='PSD WFE', index=3.0, wfe=0.25, **kwargs):
```

Comment:
It seems problematic to have the wfe parameter in units of radians, since that must be relative to some given wavelength. And below you use the wave.wavelength variable for normalizing this. If you performed a broadband calculation on this, the effect would be that the WFE (in units of nm rms) gets larger as the wavelength gets longer -  But if we think about some real mirror with polishing residuals, the total WFE on it is constant independent of wavelength. So it seems to me we need to think a bit more about how to define this normalization parameter.